### DISCUSSION TITLE
I have add function addMeetLink show error  Invalid conference type value

### DISCUSSION THREAD

**Discussion by Chiricato** [spatie/laravel-google-calendar#282] [General]
this is code
$event = new Event;

  $event->name = 'A new event';
  $event->description = 'Event description';
  $event->startDateTime = Carbon::now();
  $event->endDateTime = Carbon::now()->addHour();
  $event->addMeetLink(); // optionally add a google meet link to the event

  $event->save();

But have error
Google\Service\Exception
{ "error": { "errors": [ { "domain": "global", "reason": "invalid", "message": "Invalid conference type value." } ], "code": 400, "message": "Invalid conference type value." } }

**Comment by cardei** [comment]
Hi, same for me! Any solution ?

**Reply by alishancode** [reply]
Hey any update on this?

**Reply by jsterj** [reply]
Someone posted a potential solution on this thread: https://github.com/spatie/laravel-google-calendar/discussions/275#discussioncomment-11980580

I have not tested it out myself though. Let me know if you have any luck.

**Comment by Behram-Khattak** [comment]
Hi, I am also getting this issue, and I think there are different types for creating meetings for service account like: eventHangout and eventNamedHangout...

**Reply by hackpiy1** [reply]
The message is very misleading, and that's a problem from google's side.

You have to set a domain-wide delegation in your google workspace with the service account you already created in google console.
![Screenshot 2024-06-20 at 11 50 08 AM](https://github.com/spatie/laravel-google-calendar/assets/149876207/de46fdc9-fab7-4dce-a353-7bf1c37293a9)

**Reply by jsterj** [reply]
I have domain-wide delegation set, with the proper scope, and still receive the error. Anyone ever find a solution?

**Comment by syntx1997** [comment]
I fixed the issue by putting GOOGLE_CALENDAR_IMPERSONATE in env.
